clients/python-wrapper/tests/integration/test_reference.py

```diff
@@ -47,6 +50,19 @@ def test_reference_merge_into(setup_branch_with_commits):
     repo = lakefs.Repository(branch.repo_id)
     main = repo.branch("main")
 
+    # test merging into same branch
+
```

Comment:
You can remove the extra line

Reply:
Are there any explicit code conventions that aren't enforced by the linters?

Reply:
Generally, as with any coding project, and when working on an existing code repository we should try to stick to the following guidelines:
1. Code linters + language conventions
2. Documented coding conventions for the project
3. Adhering to the existing code structure and implicit conventions

Particularly our current comment convention is that they are always directly above the code they are documenting with no line break in between.
I also try to avoid adding unnecessary lines of code where possible (that includes newlines, redundant line breaks etc.). This might seem insignificant locally, but in the larger scheme can make the code less readable and bloated